packages/shared/src/hooks/usePostById.ts

```diff
@@ -172,7 +172,7 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
       post: post?.post,
       relatedCollectionPosts: post?.relatedCollectionPosts,
       isError,
-      isLoading: !post?.post || isLoading,
+      isLoading: !post?.post && isLoading,
```

Comment:
This will make the modals load slower I think.
Basically because I did that frontend magic of local state I only care if 1 of them doesn't exist

Reply:
Basically loading will show only when we don't have any data.

Reply:
Ah perfect in that case :)